proto/attestation/verification.proto

```diff
@@ -37,4 +37,12 @@ message AttestationResults {
 
   // Provides the reason why verification did not pass, on non-success status.
   string reason = 2;
+
+  // Contains the verified public key for encryption whenever the status
+  // indicates success. The format of the key is serialized HPKE.
+  bytes encryption_public_key = 3;
+
+  // Contains the verified public key for signing whenever the status
+  // indicates success. The format of the key is serialized ECDSA.
```

Comment:
I would suggest "The key is serialized using the SEC 1 Elliptic-Curve-Point-to-Octet-String conversion."